cleansio/explicits/explicits.py

```diff
@@ -0,0 +1,34 @@
+""" Loads list of explicits from an encrypted file """
+
+from Crypto.Cipher import AES
+import base64
+import yaml
+import os
+
+class Explicits():
+
+    def __init__(self):
+        pass
```

Comment:
Instead of `pass` do `super().__init__()`

Reply:
Why do we need `super().__init__()` here?

Reply:
I think it's good practice to always call the parent's init (even if a parent hasn't been defined yet)